README.md

```diff
@@ -1,77 +1,102 @@
 # W3bstream Sprout :four_leaf_clover:
+
 W3bstream Sprout (Alpha) which supports native Halo2 circuits (WIP) as well as zkVMs like zkWASM (WIP) and RISC0.
 
-`wsctl`` is the command line tool that interact with W3bstream protocol which can be used by node operators as well as project developers.
+`wsctl`` is the command line tool that interact with W3bstream protocol which can be used by node operators as well as
+project developers.
 
 ## Minimum requirements
 
-| Components | Version | Description |
-|----------|-------------|-------------|
+| Components                   | Version   | Description             |
+|------------------------------|-----------|-------------------------|
 | [Golang](https://golang.org) | &ge; 1.21 | Go programming language |
 
 ## Preparation
 
 ### Install docker
+
 Docker is needed to run the node service. Please make sure your docker is up to date.
+
 ```bash
 install docker & docker-compose
 ```
+
 ### Install wsctl
+
 `wsctl` is the W3bstream command line.
+
 ```bash
 curl https://raw.githubusercontent.com/machinefi/sprout/main/scripts/install_wsctl.sh | bash
 ```
 
 ## Run server
+
 ### Download docker-compose.yaml
+
 ```bash
 mkdir sprout && cd sprout
 curl https://raw.githubusercontent.com/machinefi/sprout/main/docker-compose.yaml -o docker-compose.yaml
 ```
 
 ### Populate docker-compose.yaml fields
-W3bstream-node need write proof to chain and need a private key for chain write. Set your private key at https://github.com/machinefi/sprout/blob/main/docker-compose.yaml#L20  
-If you need risc0 snark proof, a bonsai key is needed at https://github.com/machinefi/sprout/blob/main/docker-compose.yaml#L40
+
+W3bstream-node need write proof to chain and need a private key for chain write. Set your private key
+at https://github.com/machinefi/sprout/blob/main/docker-compose.yaml#L20  
+If you need risc0 snark proof, a bonsai key is needed
+at https://github.com/machinefi/sprout/blob/main/docker-compose.yaml#L40
 
 ### Use customized project code
-Docker-compose will mount current work directory to containers /data https://github.com/machinefi/sprout/blob/main/docker-compose.yaml#L23  
-So you can appoint the project file at https://github.com/machinefi/sprout/blob/main/docker-compose.yaml#L18
 
+Docker-compose will mount current work directory to containers
```

Comment:
those readme  changes  for ?

Reply:
same above, doc auto format, I will revert it